gramps/gen/datehandler/_date_nl.py

```diff
@@ -147,17 +149,17 @@ class DateDisplayNL(DateDisplay):
 
     calendar = (
         "", "juliaans", "hebreeuws",
-        "franse republiek", "persisch", "islamitisch",
+        "frans republikeins", "persisch", "islamitisch",
         "zweeds" )
 
-    _mod_str = ("", "voor ", "na ", "rond ", "", "", "")
+    _mod_str = ("", "voor ", "na ", "omstreeks ", "", "", "")
```

Comment:
Either word works here.

Reply:
Not any more, in new Dutch translation I have changed 'rond' into 'omstreeks'. So datehandler should contain 'omstreeks' too. Otherwise you could get an error when trying to fill in 'omstreeks'.
Maybe it's possible to have both words?

Reply:
Ah, keeping those two components in sync makes sense. Does this not break BC, though? E.g. will existing date entries using "rond" stop being parseable?

Reply:
Well, if you mean it doesn't change old reports or something I guess it will break BC as a translation is a 1-1 relationship.
This is a translation of the word 'about' which is translated most correctly as 'omstreeks'. The Dutch word 'rond' should be used in case the word 'around' was used and that's not the case here.
I've created a long list of words which I use now as a kind of convention for translation of all different addons.
So, maybe it breaks something, but at least it will break everywhere in same way. Hopefully the translation into Dutch will improve this way.
But, maybe it is possible to use both words by a little programming.